app/scripts/models/account.js

```diff
@@ -414,10 +416,13 @@ define(function (require, exports, module) {
         }
       })
       .then((updatedSessionData) => {
+        this._metrics.logEvent('core.signin.success');
```

Comment:
@shane-tomlinson Great summary! In addition to your changes, I think that we could add `this.logEvent('signin.success')` to the [confirm view](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/views/confirm.js#L115) if the view is waiting for a sign-in confirmation.

When doing the sign-in confirmation, the account is considered `unverified` and will go into the [if block](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/views/mixins/signin-mixin.js#L102) which redirects to the confirm_signin view and skips logging the metrics you mentioned above.

I think this would help give us one unified meaning to sign-in success. Thoughts?